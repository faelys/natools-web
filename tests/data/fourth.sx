(elements
   (main-text "\
<p>Data in the fourth page.</p>\n"
(comments
   "<h1>Timeless Comments</h1>\n<ol>\n"
   (list (render
      "<li><p>"
      (if-link "<a href=\"" (link) "\">") (if-no-link "<strong>")
      (name)
      (if-link "</a>") (if-no-link "</strong>")
      " wrote:</p>\n<blockquote>\n"
      (text)
      "</blockquote></li>\n"))
   "</ol>\n"
   (preview
      "<h1 id=\"preview\">Preview</h1>\n<ol>\n"
      "<li><p>"
      (if-link "<a href=\"" (link) "\">") (if-no-link "<strong>")
      (name)
      (if-link "</a>") (if-no-link "</strong>")
      " wrote:</p>\n<blockquote>\n"
      (text)
      "</blockquote></li>\n"
      "</ol>\n"))
"<h1 id=\"newcomment\">Post a Comment</h1>\n"
(render comment-form))
   (title "Fourth Page"))
(tags (4 pages/news tags/topic3))
(comments (backend comments 4) (tags comments) (filter comment-filter))
