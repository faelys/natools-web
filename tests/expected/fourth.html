<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Fourth Page</title>
  </head>
  <body>
<h1>Fourth Page</h1>
<p>Data in the fourth page.</p>
<h1>Timeless Comments</h1>
<ol>
</ol>
<form class="mainform" method="post" action="/fourth/comments">
  <p><label>Name or nick:
    <input name="c_name" value="" type="text">
  </label></p>
  <p><label>E-mail address:
    <input name="c_mail" value="" type="text">
  </label></p>
  <p><label>Web site:
    <input name="c_site" value="" type="text">
  </label></p><!--
  <p><label>Very special field:
    <input name="address" value="http://" type="text">
  </label></p>-->
  <p><label>Comment text:
    <textarea name="c_text" cols="80" rows="10"></textarea>
  </label></p>
  <p class="buttons">
    <input value="Preview" name="preview" type="submit">
    <input value="Submit" name="submit" type="submit">
  </p>
</form>
    <div class="sidebar">
<h1>Side bar</h1>
<p>Tags: topic3</p>
<p>Previous: Third Page</p>
    </div>
    <div class="footer">
<p>Default footer</p>
    </div>
  </body>
</html>
